gef-ui/server/controllers/select-exporters-correspondence-address/index.test.js

```diff
@@ -89,9 +92,21 @@ describe('controllers/select-exporters-correspondence-address', () => {
       );
     });
 
+    it('returns error object if address from select dropdown contains `Addresses Found`', async () => {
+      mockRequest.body.selectedAddress = 'Addresses Found';
+
+      await validateSelectExportersCorrespondenceAddress(mockRequest, mockResponse);
+
+      expect(mockResponse.render).toHaveBeenCalledWith(
+        'partials/select-exporters-correspondence-address.njk',
+        expect.objectContaining({
+          errors: expect.any(Object),
```

Comment:
should we be more specific with which error? (or is this how the rest of the tests are for similar?)

Reply:
This is how the rests of the tests are asserted. There are other instances. Not a fan, do not want to bloat this PR even more.